processor/blobuploadprocessor/metadata.yaml

```diff
@@ -0,0 +1,21 @@
+type: blobuploadconnector
+scope_name: otelcol/blobuploadconnector
+
+status:
+  class: processor
+  stability:
+    # Implementation still in-progress. Please follow along at:
+    # https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33737
+    development: [traces, logs]
+  distributions: [contrib]
+  codeowners:
+    active: [michaelsafyan, dashpole, braydonk]
+
+tests:
+  config:
+  goleak:
+    skip: false
+    ignore:
+      # See https://github.com/census-instrumentation/opencensus-go/issues/1191 for more information.
+      top: go.opencensus.io/stats/view.(*worker).start
```

Comment:
Strange that this is required given opencensus isn't in the go.sum. We can try removing it in a follow-up.